sunpy/util/config.py

```diff
@@ -13,7 +13,7 @@ def load_config():
     Read the sunpyrc configuration file. If one does not exists in the user's
     home directory then read in the defaults from module
     """
-    config = configparser.ConfigParser()
+    config = configparser.RawConfigParser()
```

Comment:
What does `Raw` do?

Reply:
It ignores the input. Apparently with normal ConfigParser you can use special sequences to add additional capability. We don't use it.